examples/cursor_position.rs

```diff
@@ -0,0 +1,50 @@
+extern crate console;
```

Comment:
Please drop the added example.

Reply:
I don't think think every tiny feature merits an example that then has to be maintained forever.